monero-outreach-docs/translations/ru/archive/wallet_primer_ru.md

```diff
@@ -0,0 +1,103 @@
+
+Резервирование также помогает избежать потери ключей. Если у вас есть две копии и вы спрятали их далеко друг от друга, локальные проблемы, подобные стихийному бедствию, будут решены.
+
+### _Заключение​_
+
+Данная статья касается искусства и науки использования бумажных кошельков Monero. Для создания ключей вам понадобится определённое программное обеспечение. Программное обеспечение кошелька, moneroaddress.org или аппаратный кошелёк позволяют сделать это. Если вам срочно необходима небольшая сумма, воспользуйтесь moneroaddress.org, но если у вас есть больше средств и вам необходим более высокий уровень защиты, используйте официальный кошелёк Monero, установленный на автономном компьютере, который никогда больше не будет подсоединён к сети. Вашему бумажному кошельку требуется по крайней мере приватный ключ траты в исходной, необработанной форме или в одной из его мнемонических форм. Этот ключ следует сохранить на физическом носителе и спрятать его каким-нибудь изощрённым способом. Вы должны позаботиться о том, чтобы не потерять свой бумажный кошелёк - создать его резервные копии и найти людей, которым будете доверять, но ограничить это доверие либо посредством некой наводящей информации, либо разделения ключа по схеме «M из N». Используя свой ключ просмотра, который вы можете хранить и передавать без риска потери Monero, вы сможете отслеживать входящие транзакции, отправленные на ваш бумажный кошелёк, с помощью публичного ключа, которым можно делиться, но ваш приватный ключ траты должен оставаться скрытым.
+
+### _Узнайте больше_
+
+- [Быстрое создание кошелька Monero](https://www.monerooutreach.org/stories/monero-wallet-quickstart.html) - Узнайте, какой кошелёк Monero подходит именно вам.
+- [Безопасный кошелек - счастливый кошелек](https://www.monerooutreach.org/monero_best_practices/a-secure-wallet-is-a-happy-wallet.html) - Лучшие практики для вашего кошелька Monero.
```

Comment:
For some reason, the Russian Monero Best Practices aren't up so I will be asking thunderosa to upload them and I will add them to the file. 

Thank you :)